Intersect.Client/Core/Graphics.cs

```diff
@@ -768,6 +812,76 @@ public static void DrawFullScreenTextureFitMaximum(GameTexture tex)
             }
         }
 
+        private static void DrawFullScreenAnimatedTexture(
+            string animFrameName,
+            int animFrameCount,
+            long animFrameSpeed
+        )
+        {
+            if (animTextureTime < Timing.Global.Milliseconds)
+            {
+                animTextureFrame += 1;
+                animTextureTime = Timing.Global.Milliseconds + animFrameSpeed;
+            }
+
+            if (animTextureFrame >= animFrameCount - 1)
+            {
+                animTextureFrame = 0;
+            }
+
+            var animTex = Globals.ContentManager.GetTexture(
+                Framework.Content.TextureType.Animation, animFrameName + "_" + animTextureFrame + ".png"
+            );
+
+            if (animTex == null)
+            {
+                return;
+            }
+
+            var destRectangle = new FloatRect();
+            var srcRectangle = new FloatRect(0, 0, animTex.Width, animTex.Height);
+            destRectangle.X = 0;
```

Comment:
Honestly the default behavior should probably be centering on the screen so you don't have an 800x600 animated background just sitting up in the top left corner of a 1080p screen

Reply:
realized that i was writing redundant code, thanks for pointing this out, i will see what i can do with a method to simply center the texture on screen as well.